gear/identifier_lookup/src/python/identifier_app/main.py

```diff
@@ -52,15 +59,19 @@ def __get_writer(self):
     def visit_header(self, header: List[str]) -> bool:
         """Prepares the visitor to write a CSV file with the given header.
 
-        If the header doesn't have `ptid`, returns an error.
+        If the header doesn't have `ptid` or `adcid`, returns an error.
 
         Args:
           header: the list of header names
         Returns:
           True if `ptid` occurs in the header, False otherwise
         """
         if FieldNames.PTID not in header:
```

Comment:
Not sure you need to change this now, but I think we should try to avoid enumerating conditionals. 

I had written the linked code to avoid repeated conditionals
https://github.com/naccdata/flywheel-gear-extensions/blob/0f4f8c6ae3bf03bbb5f31054af00fb1012594ad3/gear/identifier_provisioning/src/python/identifier_provisioning_app/main.py#L416